src/Components/Views/Profile/Profile.js

```diff
@@ -1,24 +1,100 @@
 import React, { Component } from 'react';
 import styled from 'styled-components';
+import Cookies from 'universal-cookie';
 
+//Inits cookies
+const cookies = new Cookies();
 
 const ScProfile = styled.div`
-    text-align: center;
     display: flex;
-    flex-direction: column;
-    align-items: center;
-    justify-content: center;
     font-size: 24px;
+    
+    margin: auto;
+    text-align: center;
+    justify-content: center;
+    margin-top: 4%;
 `;
 
+const ScInput = styled.input`
+`;
 
 class Profile extends Component {
+    constructor(props){
+
+        super(props)
+        this.state = {
+            blank: false,
+        }
+    }
+
+    //Handle Change of User's Name
+    handleNameChange = e => {
+        cookies.set('username', e.target.value);
+    }
+
+    //Handle Change of User's Allergy
+    handleAllergyChange = e => {
+        cookies.set('allergy', e.target.value);
+    }
+
+    //Handle Change of User's Favorite Foods
+    handleFavFoodChange = e => {
+        cookies.set('favFood', e.target.value);
+    }
+
+    //Handle Change of User's Least Favorite Foods
+    handleNFavFoodChange = e => {
+        cookies.set('nfavFood', e.target.value);
+    }
+
+    //Handle Change of User's Favorite Restauraunt
+    handleRestChange = e => {
+        cookies.set('favRest', e.target.value);
+    }
+
+    //Handle Submission of the user data form
+    handleSave = e => {
+        this.setState({blank:false});
+    }
+
+    //Handle Clicking of edit button
+    handleEdit = e => {
+        this.setState({blank:true});
+    }
+
     render() {
-        return (
-            <ScProfile>
-                This is ( ͡° ͜ʖ ͡°)
-            </ScProfile>
-        );
+        if(this.state.blank == true){
```

Comment:
better practice to use === instead of ==.
Also is this supposed to be === true or !== true? (im not saying youre wrong im just checking)